__mocks__/react-i18next.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { translate, translateWithFormatting } from '@/utils/test/i18n';
```

Comment:
@JQrdan @matthew-chirgwin - https://react.i18next.com/latest/usetranslation-hook - we can either use named export `t` or we can use the `[t, i18next]` version and rename it on every import.

From mock perspective, I'll update it to return both APIs for safety. But I'll have to export `t` as function regardless.